spec/controllers/internal_api/v1/pillars_controller_spec.rb

```diff
@@ -309,7 +309,8 @@
             floating:       "9bc3e819-a6ca-648b-b5e3-c26c9e6c5e57",
             subnet:         "4b64b38d-0b38-40d0-a69f-ade7299ef4ab",
             bs_version:     "v2",
-            lb_mon_retries: "3"
+            lb_mon_retries: "3",
+            ignore_vol_az:  "false"
```

Comment:
Is there a reason for this to be a string instead of a boolean? The JSON is going to be a little unnatural as a string